### DISCUSSION TITLE
2023-10-04

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#612] [daily_meeting]
### 오늘의 할 일
공통
- 오전 강의

백엔드
- 오후 리뷰데이

프론트엔드
- 각자 작업 진행

### 홍보 관련
- 데분당태 홍보 관련으로 왼손에게 답장받고 채팅 보내기 - 테오
- 새로운 메일 답변 쓰기 - 룩소

### 내일의 할 일

**Comment by yeopto** [comment]
### 체크인

10점 - 재택 너무 좋아요

### 체크아웃

8점 - 워케이션 여정은 험난해요

**Comment by woosung1223** [comment]
### 체크인
7점 - 사다리타기 억까가 좀 심하네요

### 체크아웃
7점 - 재택이 역시 좋은 것 같습니다..

**Comment by aak2075** [comment]
### 체크인

10점 - 삶의 질 상승

**Comment by jaehee329** [comment]
### 체크인
9점 - 과연 집에서 시간 활용 잘 할 수 있을까?
### 체크아웃
10점 - 못할거같아서 나와서 잘 하고 있네요

**Comment by woo-jk** [comment]
### 체크인
10점 - 이래서 재택 재택 하는구나

### 체크아웃
10점 - 시간이 빨리가요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 연휴때 기능 구현은 다해서 좋은데 미션이랑 리뷰랑 할게 좀 많네요... 

### 체크아웃
9점 - 이제 리뷰는 다 했으니 리뷰반영하고 미션만하면...!

**Comment by nlom0218** [comment]
### 체크인

7점 - 일어나서 아무것도 안했네요.

### 체크아웃

10점 - 달력 PR 올렸으니..!
